src/tests/ftest/util/soak_utils.py

```diff
@@ -712,10 +712,9 @@ def start_dfuse(self, pool, container, name=None, job_spec=None):
             cmd(list):          list of dfuse commands to add to job script
     """
     # Get Dfuse params
-    dfuse = Dfuse(self.hostlist_clients, self.tmp)
-    dfuse.namespace = os.path.join(os.sep, "run", job_spec, "dfuse", "*")
-    dfuse.bind_cores = self.params.get("cores", dfuse.namespace, None)
```

Comment:
I think you still need to get `bind_cores` here. `get_dfuse` does not, but `start_dfuse` does. (separate issue we need to address in the framework)

Reply:
thanks for catching that